src/App.js

```diff
@@ -1,5 +1,94 @@
+import { Console, MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async run() {}
+  async run() {
+    const carNameString = await Console.readLineAsync(
+      "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n"
+    );
+    let carNames = carNameString
+      .split(",")
+      .map((carName) => this.validateName(carName));
+    carNames = this.validateDuplicateName(carNames);
+
+    let playNum = await Console.readLineAsync("시도할 횟수는 몇 회인가요?\n");
+    playNum = this.isPositiveNumber(playNum);
+
+    const racingCars = carNames.map((carName) => {
+      return { name: carName, score: 0 };
+    });
+
+    Console.print("\n실행 결과");
+    for (let i = 0; i < playNum; i++) {
+      racingCars.map((car) => {
+        this.randomForward(car);
+        this.printRacingcarInfo(car);
+      });
+      Console.print("\n");
+    }
+
+    Console.print(
+      `최종 우승자 : ${this.getWinner(racingCars)
+        .map((car) => car.name)
+        .join(", ")}`
+    );
+  }
+
+  randomForward(car) {
+    if (MissionUtils.Random.pickNumberInRange(0, 9) >= 4) {
+      car.score += 1;
+    }
+  }
+
+  printRacingcarInfo(car) {
+    Console.print(`${car.name} : ${"-".repeat(car.score)}`);
+  }
+
+  getWinner(racingCars) {
+    let winner = [];
```

Comment:
```suggestion
    const winner = [];
```
getWinner에서 winner를 리턴하고 있으니 let이 아닌 const를 사용하는 건 어떨까요??